packages/strapi-plugin-upload/admin/src/containers/SettingsPage/index.js

```diff
@@ -111,6 +112,8 @@ const SettingsPage = () => {
 
   return (
     <>
+      <Helmet title="Settings - Media Library" />
```

Comment:
Don't we need to use translated messages for page title?

Reply:
ooh gosh! totally forgot 🤦🏻 . Thanks !

Reply:
😂